File: vivo-frontend/src/app/models/BacDto.ts
import {Product} from "./Product";

export class BacDto{
  idBac: String|undefined;
  capacity: number|undefined;
  totalImpom: number|undefined;
  status: boolean|undefined;
  dateOuverture: Date|undefined;
  capacityUsed: number|undefined;
  idProduct: number|undefined;
  product?: Product;

}

File: Stock/src/main/java/net/lahlalia/stock/dtos/BacDto.java
package net.lahlalia.stock.dtos;

import lombok.*;

import java.util.Date;

@Getter
@Setter
@Builder // for building objects using .build
@AllArgsConstructor
@NoArgsConstructor
public class BacDto {

    private String idBac;
    private double capacity;
    private double totalImpom;
    private boolean status;
    private Date dateOuverture;
    private double capacityUsed;
    private Long idProduct;
    private String idDepot;

}
